src/releng_docs/index.rst

```diff
@@ -34,7 +34,7 @@ Install Nix:
 
 .. code-block:: bash
 
-    % curl https://raw.githubusercontent.com/mozilla-releng/services/master/nix/setup.sh | sh
+    % curl https://raw.githubusercontent.com/mozilla-releng/services/master/nix/setup.sh | bash
```

Comment:
also above comment should not be a stopper to merged this, since majority use debian/ubuntu